frontend/appflowy_flutter/lib/plugins/document/presentation/editor_drop_handler.dart

```diff
@@ -37,12 +37,34 @@ class EditorDropHandler extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final childWidget = Consumer<EditorDropManagerState>(
-      builder: (context, dropState, _) => DropTarget(
-        enable: dropState.isDropEnabled,
-        onDragExited: (_) => editorState.selectionService.removeDropTarget(),
-        onDragUpdated: _onDragUpdated,
-        onDragDone: _onDragDone,
-        child: child,
+      builder: (context, dropState, _) => DragTarget<ViewPB>(
+        onLeave: (_) => editorState.selectionService.removeDropTarget(),
+        onMove: (details) {
+          if (details.data.id == viewId) {
```

Comment:
Do we still need to skip the move action here when returning false in the `onWillAcceptWithDetails` function?

Reply:
Yes, the `onMove` is called no matter the result of `onWillAcceptWithDetails`, I also assumed that returning false there would be enough to cancel auxiliary callbacks, but it's not.

But we can just return here, no need to remove drop target, let me fix.